src/components/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
```

Comment:
remove app.jsx from components you're changing vite folder structure, and even app.jsx is not components